### PR TITLE
replace wasm for running project

### PR DESCRIPTION
**Pull request by hunshenshi**

### CODE REVIEW

File: cmd/srv-applet-mgr/apis/applet/put.go [hunk 1]
```diff
@@ -11,22 +11,19 @@ import (
...
 
 func (r *UpdateApplet) Output(ctx context.Context) (interface{}, error) {
 	ca := middleware.CurrentAccountFromContext(ctx)
 
-	app, err := applet.GetAppletByAppletID(ctx, r.AppletID)
+	ctx, err := ca.WithAppletContext(ctx, r.AppletID)
 	if err != nil {
 		return nil, err
 	}
-	if _, err = ca.ValidateProjectPerm(ctx, app.ProjectID); err != nil {
```
Comment (CoderZhi) [inline cmd/srv-applet-mgr/apis/applet/put.go:27]: why could the checking be skipped?

File: pkg/modules/vm/instance_new.go [hunk 2]
```diff
@@ -2,16 +2,22 @@ package vm
...
 	"context"
+	"github.com/machinefi/w3bstream/pkg/enums"
 	"os"
 
 	"github.com/machinefi/w3bstream/pkg/modules/vm/wasmtime"
 	"github.com/machinefi/w3bstream/pkg/types"
 )
 
 func NewInstance(ctx context.Context, path string, id types.SFID) error {
+	NewInstanceWithState(ctx, path, id, enums.INSTANCE_STATE__CREATED)
```
Comment (CoderZhi) [inline pkg/modules/vm/instance_new.go:13]: return

File: cmd/srv-applet-mgr/apis/applet/put.go (hunk 1 above)
Comment (hunshenshi) [reply cmd/srv-applet-mgr/apis/applet/put.go:27]: include in the `ca.WithAppletContext`

File: pkg/modules/vm/instance_new.go [hunk 3]
```diff
@@ -2,16 +2,21 @@ package vm
 
 import (
 	"context"
+	"github.com/machinefi/w3bstream/pkg/enums"
```
Comment (CoderZhi) [inline pkg/modules/vm/instance_new.go:5]: wrong group

**Review by CoderZhi — APPROVED** [review]

**Pull request merged**